src/main/java/com/courtcanva/ccfranchise/models/Order.java

```diff
@@ -0,0 +1,79 @@
+package com.courtcanva.ccfranchise.models;
+
+
+import com.courtcanva.ccfranchise.constants.EmployeeStatus;
+import com.courtcanva.ccfranchise.constants.OrderStatus;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+import org.hibernate.annotations.UpdateTimestamp;
+import java.math.BigDecimal;
+import java.time.OffsetDateTime;
+import javax.persistence.*;
+
+import com.vladmihalcea.hibernate.type.json.JsonBinaryType;
+
+@Entity
+@Setter
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "`order`")
+@TypeDef(name = "jsonb", typeClass = JsonBinaryType.class)
+public class Order {
```

Comment:
where is the related db migration script?

Reply:
After discussion with Gordon, the order entity has been modified, and its db migration script is in V1__init_tables.sql.